cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -737,23 +737,23 @@ features:
         name-label: "Name"
         name-placeholder: "Namespace name"
         scheduler-queue-label: "Scheduler Queue"
-        scheduler-queue-placeholder: "Specify the YARN queue name to be associated with this namespace"
+        scheduler-queue-placeholder: "Specify the YARN queue name to be used for programs in this namespace"
         sld-desc:  "Specify the name and the description of the namespace."
         ssd-label: "General Information"
       Step2:
         hbase-nm-name-label: "HBase Namespace Name"
-        hbase-nm-name-placeholder: "Existing namespace in HBase for new namespace"
+        hbase-nm-name-placeholder: "Namespace in HBase for datasets in the new namespace"
         hdfs-root-directory-label: "HDFS Root Directory"
-        hdfs-root-directory-placeholder: "Root directory of HDFS for new namespace root"
+        hdfs-root-directory-placeholder: "Base directory on HDFS for the new namespace"
         hive-db-name-label: "Hive Database Name"
-        hive-db-name-placeholder: "Hive db name for the new namespace."
-        sld-label: "Specify mapping of namespace resources to the underlying existing storage resources."
+        hive-db-name-placeholder: "Hive database for the new namespace"
+        sld-label: "Specify mapping of namespace resources to the existing underlying storage resources."
         ssd-label: "Namespace Mapping"
       Step3:
         keytab-uri-label: "Keytab URI"
         keytab-uri-placeholder: "Location of keytab file associated with the principal"
         principal-label: "Principal"
-        principal-placeholder: "User to be impersonated-as for jobs in the new namespace"
+        principal-placeholder: "User to be impersonated-as for programs in the new namespace"
```

Comment:
the user is used for running programs. This does not specify where it is configured, just where it is used.